src/views/footerView.jsx

```diff
@@ -72,10 +72,17 @@ export const ConferenceFooter = ({ isChatOpen, toggleChat }) => {
   };
   const [errorModal, setErrorModal] = useState(initialModalProps);
 
-  async function startPlugin() {
-    if (!pluginRef.current) {
+  function createVBPlugin() {
+    if(!pluginRef.current){
       pluginRef.current = new HMSVirtualBackgroundPlugin("none");
     }
+  }
+
+  async function startPlugin() {
+    if(!pluginRef.current){
```

Comment:
The check can be removed.

Reply:
It is required because we are making it to null in removePlugin()

Reply:
The check is there in createVBPlugin function which can be called directly here.

Reply:
yeah that is also correct